src/app/stacktrace_win.h

```diff
@@ -91,7 +91,7 @@ BOOL CALLBACK straceWin::EnumModulesCB(LPCSTR ModuleName, DWORD64 BaseOfDll, PVO
 {
     Q_UNUSED(ModuleName)
     IMAGEHLP_MODULE64 mod;
-    EnumModulesContext* context = (EnumModulesContext*)UserContext;
+    auto *context = static_cast<EnumModulesContext *>(UserContext);
```

Comment:
https://github.com/qbittorrent/qBittorrent/pull/15240/files#r678188071

> You don't need to add `*` to `auto` if the right part clearly expresses that it is a pointer. This only improves readability in less obvious cases, for example.: `auto session = BitTorrent::Session::instance();`

Also here.

Reply:
I expect that the author will extend the effect of my comment to all such cases in the current PR (at least, I am sure that I do not need to point out to @Chocobo1 every such case).

Reply:
@glassez 

Best to be explicit about it. You could also have mentioned in your other comment "there are other similar instances", instead of pointing it out everywhere (which I agree can be tiresome), as I've seen you do in the past.